### PR TITLE
upgrade to vega 3.0, vega-lite 2.0

### PR DESCRIPTION
**Pull request by sirahd**
- Update `vega-view` API to match the most current version
- Update `vega` and `vega-lite` examples specs to match the recent version

### CODE REVIEW

**Comment by sirahd** [comment]
@kanitw please see if there's anything wrong, otherwise I'll move on to fixing the revert label in tooltip.

**Comment by kanitw** [comment]
@sirahd  Why are there some conflicts?  You might wanna rebase to master first?

**Comment by sirahd** [comment]
@kanitw Done!

**Comment by domoritz** [comment]
- [ ] Do not include the compiled files in git (add `build` to .gitignore)
- [ ] Move the css file back out of build but add a step to the build script to copy it.

File: examples.js [hunk 1]
```diff
@@ -66,17 +65,8 @@
...
-    },
-    onDisappear: function() {
-      console.log("tooltip disappears!");
-    },
-    onMove: function() {
-      console.log("tooltip moves!");
-    }
   }
-  addVlExample("exampleSpecs/bubble_multiple_aggregation.json", "#vis-bubble-multi-aggr", bubbleOpts);
+  // addVlExample("exampleSpecs/bubble_multiple_aggregation.json", "#vis-bubble-multi-aggr", bubbleOpts);
```
Comment (domoritz) [inline examples.js]: What's up with these commented out lines? Please delete them if they are not needed anymore.

File: src/index.ts [hunk 2]
```diff
@@ -83,7 +83,7 @@ export function vega(vgView: VgView, options: Option = {}) {
 * It can also provide custom title and format for fields
 * options can be supplemented by vlSpec
 */
-export function vegaLite(vgView: VgView, vlSpec: Spec, options: Option = {}) {
+export function vegaLite(vgView: VgView, vlSpec: TopLevelExtendedSpec, options: Option = {}) {
```
Comment (domoritz) [inline src/index.ts]: unfortunately this is going to change so we cannot make a release yet

File: package.json [hunk 3]
```diff
@@ -2,14 +2,14 @@
   "name": "vega-tooltip",
   "version": "0.1.3",
```
Comment (domoritz) [inline package.json]: Update version to the 1.0 beta

File: src/index.ts (hunk 2 above)
Comment (sirahd) [inline src/index.ts]: So should I change it back to `Spec`? But that's gonna cause an error because of mismatch type.

File: examples.js (hunk 1 above)
Comment (sirahd) [inline examples.js]: I put it on when testing and forgot to remove it back

File: src/index.ts (hunk 2 above)
Comment (domoritz) [inline src/index.ts]: No leave it for now. Will fix it later.

**Comment by domoritz** [comment]
- [ ] Do not include all of vega-lite in the output

**Pull request merged**